pysmurf/tune/smurf_tune.py

```diff
@@ -2628,7 +2453,8 @@ def unset_fixed_flux_ramp_bias(self,acCouple=True):
                 time.sleep(self._cryo_card_relay_wait)
 
 
-    def set_fixed_flux_ramp_bias(self,fractionFullScale,debug=True, do_config=True):
+    def set_fixed_flux_ramp_bias(self,fractionFullScale,debug=True, d
```

Comment:
Typo?

Reply:
Oh I see you caught this one already in the last commit in this group.  Resolved.